get-results.sh

```diff
@@ -3,22 +3,30 @@
 HOST="127.0.0.1"
 PORT="3000"
 
+processJob () {
```

Comment:
Duplicated code moved to separate function that we can call from the other places.